readthedocs/api/v3/mixins.py

```diff
@@ -92,3 +100,20 @@ def get_queryset(self):
 
         # List view are only allowed if user is owner of parent project
         return self.listing_objects(queryset, self.request.user)
+
+
+class UpdatePartialUpdateMixin:
```

Comment:
This is a super confusing name. Why does it have `Update` in the name twice?

Reply:
I should probably rename this to `UpdateMixin` only. This class makes PUT to return 204 on success as PATCH.